web/js/main.jsx

```diff
@@ -183,7 +185,7 @@ return(<div className="custom-button waitingroom">{player}</div>)
 
   login(type){
 
-    const socket = type==='host' ? `wss://${location.host}/ws?name=${this.state.playerName}` : `wss://${location.host}/ws?name=${this.state.playerName}&game=${this.state.gameId}`
+    const socket = type==='host' ? `ws://${location.host}/ws?name=${this.state.playerName}` : `ws://${location.host}/ws?name=${this.state.playerName}&game=${this.state.gameId}`
```

Comment:
swap back to wss

We should explore having webpack provide environment variables so that we can just say:

```
if (process.env.NODE_ENV === 'production){ /* use wss */ } else { /* use ws */ }
```